app/views/admin/settings/appearance/show.html.haml

```diff
@@ -22,6 +22,31 @@
               value_method: ->(value) { value.join('/') },
               group_method: :last
 
+  .fields-row
+    .fields-row__column.fields-row__column-6.fields-group
+      = f.input :system_dark,
+                collection: Themes.instance.flavours_and_skins.each { |(_, fsa)| fsa.delete_if { |fs| fs[1] == 'system' } },
```

Comment:
This bothers me, but `flavour_and_skins` format requires this to remove the system skin from the options.
The only other way would be to have a different array which excludes "system", but it would have no use anywhere else.